### ISSUE TITLE
Coding style is not documented

### ISSUE THREAD

**Issue by sbc100** [WebAssembly/wabt#1273]
Ran into this issue when reviewing some recent additions.

Currently the only documentation I could find was that we specify `BasedOnStyle: Chromium` in `.clang-format`.   This implicitly locks down a lot of things which good, more explicit docs would be good.

The specific issue we ran into was the wabt uses a style that requires curly braces around conditional blocks, but chromium, which inherits from google style allows from single line conditional, and conditional without curlys:
https://google.github.io/styleguide/cppguide.html#Conditionals
`In general, curly braces are not required for single-line statements`

We should be specific if we want to enforce this style.

**Comment by sbc100** [comment]
@binji @aardappel

**Comment by binji** [comment]
Yeah, I had a warning for missing curly braces I think, but maybe it's not triggering anymore...? In any case, it would be better to be clear about this. However, AFAIK it's not possible in the `.clang-format` file, since it generally only adds/removes whitespace.

**Comment by sbc100** [comment]
What do you think about relaxing the style to match chomium's :)

**Comment by sbc100** [comment]
Should we add a paragraph about the style guidelines to `Contributing.md`?

**Comment by binji** [comment]
IIRC, I switched to requiring braces because it seemed like most folks preferred it.

Contributing.md seems like a reasonable place, yeah.
